src/cargo/util/toml/mod.rs

```diff
@@ -1041,6 +1041,43 @@ impl<T> MaybeWorkspace<T> {
     }
 }
 
+fn workspace_vec_string<'de, D>(
+    deserializer: D,
+) -> Result<Option<MaybeWorkspace<Vec<String>>>, D::Error>
+    where
+        D: de::Deserializer<'de>,
+{
+    struct Visitor;
+
+    impl<'de> de::Visitor<'de> for Visitor {
+        type Value = Option<MaybeWorkspace<Vec<String>>>;
+
+        fn expecting(&self, formatter: &mut fmt::Formatter<'_>) -> fmt::Result {
+            formatter.write_str("an array")
```

Comment:
```suggestion
            formatter.write_str("vector of strings")
```

Reply:
array would be more of the correct toml term

Reply:
I copied it from other error messages. Maybe we should unify it.